netjsonconfig/backends/base.py

```diff
@@ -256,7 +277,10 @@ def render(self):
 
     def get_context(self):
         """
-        Builds context that is passed to jinja2 templates
+        Builds context dictionary to be used in jinja2 templates
+        For every getter method suche as ``_get_name`` creates an entry in
```

Comment:
`suche` should be `such`

`getter` and `setter` methods are a slightly different thing in python (see [descriptor pattern](https://docs.python.org/3/howto/descriptor.html) for more information).

Something like the following would be more appropriate:
"For every method prefixed with ``_get_``, such as ``_get_name``, ..."